libs/template/templates/default-sql/template/{{.project_name}}/scratch/README.md

```diff
@@ -0,0 +1,4 @@
+# scratch
+
+This folder is reserved for personal, exploratory notebooks.
```

Comment:
notebooks and SQL files.

Reply:
Maybe just talk about SQL files here. If we wanted to support notebooks then we should be adding a `requirements.txt` to configure the python env.

Reply:
I added SQL files. And notebooks are actually very useful for SQL at this point. They can be used with warehouses too (though probably not in the IDE, yet?)

Re. requirements.txt: is that used for notebooks?

Reply:
Notebooks in the IDE currently use `spark.sql` so it's technically not the same. I could imagine adding a SQL mode to the notebooks just like in the webapp but that's currently not scoped.

Notebooks can be used without virtual envs but that would clutter the global Python module space. If we recommend notebooks then using virtual envs and having a requirements.txt would be best practice.